### PR TITLE
cc31

### PR DESCRIPTION
**Pull request by MarikoAlvarado**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/repeatedWord/reapeated-word.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+function findWord(str) {
+  let index = 0
+  while (str[index]) {
+    let val = str[index];
+    console.log(val)
+    index++;
+  }
+}
+findWord(str);
```
Comment (adrianhuebner) [inline javascript/code-challenges/repeatedWord/reapeated-word.js:104]: This is causing some issues, I am getting an error when I run this code that says str isn't defined.
One thing as a tip if you want to do multiple module.export you can write it like this:
```
module.exports = [ HashMap, findWord ]
```
At the bottom of the page, for less code being wrtitten!
